_data/packages.yml

```diff
@@ -1,18 +1,57 @@
+- package_name: THzTools
+  package_description:
+  submitting_author:
+      name: Name
+      github_username: jsdodge
+  all_current_maintainers:
+    - name: J. Steven Dodge
+      github_username: jsdodge
+  repository_link: https://github.com/dodge-research-group/thztools
+  version_submitted: 0.5.4
+  categories:
+    - data-processing-munging
+  editor:
+      name: Bane Sullivan
+      github_username: banesullivan
+  eic:
+      name:
+      github_username: cmarmo
+  reviewers:
+    - name: Frank Male
+      github_username: frank1010111
+    - name:
+      github_username: Romain-Peretti
+  archive: TBD
+  version_accepted: TBD
+  date_accepted: missing
+  created_at: 2024-08-01 11:54:53+00:00
+  updated_at: 2024-11-20 00:07:21+00:00
+  closed_at:
+  issue_link: https://github.com/pyOpenSci/software-submission/issues/209
+  joss:
+  partners:
+  gh_meta:
+      name: thztools
+      description: Tools for terahertz time-domain spectroscopy (THz-TDS)
+      created_at: '2022-11-22'
+      stargazers_count: 10
+      watchers_count: 10
+      open_issues_count: 2
+      forks_count: 1
+      documentation: https://dodge-research-group.github.io/thztools/
+      contrib_count: 5
+      last_commit: '2024-11-19'
+  labels:
+    - 6/pyOS-approved
 - package_name: Stingray
   package_description: A spectral-timing software package for astrophysical X-ray
       (and other) data
   submitting_author:
-      name: Matteo Bachetti
+      name: Name
       github_username: matteobachetti
   all_current_maintainers:
     - name: Daniela Huppenkothen
-      github_username: dhuppenkothen
-    - name: Guglielmo Mastroserio
-      github_username: mgullik
-    - name: John Swinbank
-      github_username: jdswinbank
-    - name: Matteo Lucchini
-      github_username: matteolucchini1
+      github_username: dhuppenkothen mgullik jdswinbank matteolucchini1
```

Comment:
This is problematic list of usernames. I suspect this is happening because there are no commas separating the names so pyosmeta is parsing it all as a string. however, sometimes we have people add their name, @github username. maybe we parse on the @ instead of the commas??

Reply:
the alternative here is to be specific about the format we want this data in.

Reply:
this has been fixed ad should be updated when we do a release